package.json

```diff
@@ -21,7 +21,6 @@
     "appId": "org.jitsi.jitsi-meet",
     "productName": "Jitsi Meet",
     "generateUpdatesFilesForAllChannels": true,
-    "afterSign": "./notarize.js",
```

Comment:
Oh, so it now works out of the box? Great!

Reply:
Yes, it does :)